ReactiveObjC/RACSourceSymbolExtractor.m

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2021 Lightricks. All rights reserved.
+// Created by Ofir Gluzman.
+
+#if DEBUG
+
+#import "RACSourceSymbolExtractor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSString *RACExtractSourceSymbol(NSArray<NSString *> *callStackSymbols) {
+  static NSString * const kPattern = @"[0-9]+[\\s]+([\\S]+)[ ]+0x[0-9,a-f]+ (.*?) \\+ ([0-9]+)";
+  static NSString * const kRACLibraryName = @"ReactiveObjC";
+
+  NSError *error;
+  NSRegularExpression * _Nullable regex = [NSRegularExpression
```

Comment:
Since the regular expression is global, I suggest to initialize this once as well since compiling a regex is a heavy operation.

Reply:
irrelevant - after moving to dladdr